src/plugins/messenger/MessengerPreview/components/MessengerMediaTemplate/MessengerMediaTemplate.tsx

```diff
@@ -1,94 +1,95 @@
-import ReactPlayer from 'react-player';
```

Comment:
please revert undesired styling changes. E.g.: This file now has tab stops of 2 spaces, we always use 4

Reply:
fixed